operator/pkg/ciliumendpointslice/testutils/no_op_workqueue_metrics_provider.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package testutils
+
+import "k8s.io/client-go/util/workqueue"
+
+// NoOpWorkqueueMetricsProvider is a no-op implementation of workqueue.MetricsProvider.
+type NoOpWorkqueueMetricsProvider struct{}
```

Comment:
Is this actually necessary? I would expect that just passing a nil value preserves the current behavior.

Reply:
Right, I already did this in another place, but completely forgot here for some reason 🤦